drop-in/src/main/java/com/adyen/checkout/dropin/ui/paymentmethods/PaymentMethodsListViewModel.kt

```diff
@@ -110,6 +106,22 @@ class PaymentMethodsListViewModel(
         }
     }
 
+    private fun PaymentMethod.mapToModel(): PaymentMethodModel {
+        val icon = when (type) {
+            PaymentMethodTypes.SCHEME -> CARD_LOGO_TYPE
+            PaymentMethodTypes.GOOGLE_PAY -> GOOGLE_PAY_LOGO_TYPE
```

Comment:
This might be a bit tricky, I think the TxVariant itself will also change from `paywithgoogle` to `googlepay`, so this string would actually be the payment method type, at least that's what I got from COAND-397.

Maybe for this PR it's fine like this, but then when implementing COAND-397 we would probably make `PaymentMethodTypes.PAY_WITH_GOOGLE`. Do you have more info on how this will reflect on the API changes?

Reply:
Yep, I wasn't aware of the new payment method type, but I prefer to fix it in the next PR, just to keep it in context, right?